tests/imports.py

```diff
@@ -64,8 +63,9 @@ def test_tools(self):
             "deploy_process",
             "LocalPathExpander",
             "get_module",
+            "hdmf",
         ]
-        self.assertCountEqual(first=current_structure, second=expected_structure)
+        assert sorted(current_structure) == sorted(expected_structure)
```

Comment:
An example of the new error message

```python
E       AssertionError: assert ['LocalPathExpander', 'deploy_process', 'get_module', 'get_package', 'hdmf', 'importing', 'nwb_helpers', 'path_expansion', 'processes', 'yaml_conversion_specification'] == ['LocalPathExpander', 'deploy_process', 'get_module', 'get_package', 'importing', 'nwb_helpers', 'path_expansion', 'processes', 'yaml_conversion_specification']
E         At index 4 diff: 'hdmf' != 'importing'
E         Left contains one more item: 'yaml_conversion_specification'
E         Full diff:
E           [
E            'LocalPathExpander',
E            'deploy_process',
E            'get_module',
E            'get_package',
E         +  'hdmf',
E            'importing',
E            'nwb_helpers',
E            'path_expansion',
E            'processes',
E            'yaml_conversion_specification',
E           ]
```

compared to the previous

```python
>       self.assertCountEqual(first=current_structure, second=expected_structure)
E       AssertionError: Element counts were not equal:
E       First has 1, Second has 0:  'hdmf'
```

and also paves the way for these to be pure `pytest` instead of `unittest.TestCase`